lib/tokenize.cpp

```diff
@@ -2709,7 +2709,7 @@ void Tokenizer::setVarIdPass1()
             if (!isC() && Token::simpleMatch(tok2, "const new"))
                 continue;
 
-            bool decl;
+            bool decl = false;
```

Comment:
this initialization seems redundant. The syntaxError throws an exception as far as I know.

Reply:
You are right.  Yes it does.